docs/contributing/dependencies/index.md

```diff
@@ -0,0 +1,116 @@
+---
+sidebar_custom_props:
+  access: bitwarden
+---
+
+# Dependency Management
+
+Bitwarden uses [Renovate](https://www.mend.io/renovate/) for automating dependency updates. Renovate
+will automatically create pull requests for dependencies on a weekly cadence. Security updates will
+generate pull requests immediately.
+
+## Ownership
+
+Bitwarden's repositories falls under two categories, team-owned and shared repositories.
+
+### Team-Owned repositories
+
+Team owned repositories are "owned" by a single team from a dependency standpoint. The assigned team
+is responsible for reviewing, approving and merging dependency updates. Some reason a repository
```

Comment:
```suggestion
is responsible for reviewing, approving and merging dependency updates. Sometimes a repository
```

Reply:
I don't think `Sometimes` convey the same meaning here. `Sometimes a repository might be team-owned are that it's primarily` doesn't make much sense.

Reply:
This sentence might have become reworded, can you verify if you still want `Sometimes` here? I don't think it fits in any longer.